src/LudoGameApp/GameEngine/LudoEngine.cs

```diff
@@ -0,0 +1,280 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+namespace GameEngine
+{
+    public class LudoEngine
+    {
+        public int Counter = 0;
```

Comment:
What does this count? The naming does not reveal what is used for.
Does it need to be public?